packages/units/src/unitsProvider.js

```diff
@@ -23,7 +23,7 @@ import PropTypes from 'prop-types';
 /**
  * Internal dependencies
  */
-import StoryPropTypes from '../types';
+import StoryPropTypes from '../../../assets/src/edit-story/types';
```

Comment:
We must avoid any dependencies on `edit-story` in this package.

In the experimental branch I just solved this by defining the prop types here like this:

```js
UnitsProvider.propTypes = {
  pageSize: PropTypes.exact({
    width: PropTypes.number.isRequired,
    height: PropTypes.number.isRequired,
  }).isRequired,
  children: PropTypes.node.isRequired,
};
```

Reply:
Fixed.